src/Common/test-runtime/project.json

```diff
@@ -40,11 +40,13 @@
     "System.Runtime.Extensions": "4.0.10",
     "System.Runtime.Handles": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
+	"System.Runtime.InteropServices.RuntimeInformation": "4.0.0-rc3-23725",
```

Comment:
I thought this was not available for NET Native.  Also nit: indent is different from others.